### PR TITLE
updated default list of modules in poller and discovery support docs.

### PR DESCRIPTION
**Pull request by barryodonovan**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

The list of en/disabled poller and discovery modules in the support docs was out of date.
They are now current as per ./include/defaults.inc.php

### CODE REVIEW

File: doc/Support/Poller Support.md [hunk 1]
```diff
@@ -42,38 +42,49 @@ These are the default poller config items. You can globally disable a module by
...
+$config['poller_modules']['netstats']                    = 1;
+$config['poller_modules']['hr-mib']                      = 1;
+$config['poller_modules']['ucd-mib']                     = 1;
+$config['poller_modules']['ipSystemStats']               = 1;
+$config['poller_modules']['ports']                       = 1;
+$config['poller_modules']['bgp-peers']                   = 1;
+$config['poller_modules']['junose-atm-vp']               = 0;
+$config['poller_modules']['toner']                       = 0;
+$config['poller_modules']['ucd-diskio']                  = 1;
+$config['poller_modules']['wifi']                        = 0;
```
Comment (murrant) [inline doc/Support/Poller Support.md]: Can you remove wifi?  It is deprecated.

Comment (barryodonovan) [reply doc/Support/Poller Support.md]: It is still in the defaults file, should it not remain consistent until wifi is obsolete?

Comment (laf) [reply doc/Support/Poller Support.md]: It's still doing stuff though so should be kept until it is deprecated.

File: doc/Support/Poller Support.md [hunk 2]
```diff
@@ -42,43 +42,54 @@ These are the default poller config items. You can globally disable a module by
...
+$config['poller_modules']['aruba-controller']            = 0;
+$config['poller_modules']['entity-physical']             = 1;
+$config['poller_modules']['applications']                = 1;
+$config['poller_modules']['mib']                         = 0;
+$config['poller_modules']['stp']                         = 1;
+$config['poller_modules']['ntp']                         = 1;
+$config['poller_modules']['services']                    = 1;
+$config['poller_modules']['loadbalancers']               = 0;
+$config['poller_modules']['mef']                         = 0;
+$config['poller_modules']['tnms-nbi']                    = 0;
```
Comment (laf) [inline doc/Support/Poller Support.md]: I think this line can go, tnms-nbi isn't a poller module.

**Comment by barryodonovan** [comment]
OK, it is in the defaults file though, I just C&P from the file. I was trying to think of an easy way to just include the section of the file into the docs but more hassle than it's worth.
________________________________

**Comment by laf** [comment]
Can be deleted from the defaults as well then :D

File: doc/Support/Poller Support.md (hunk 1 above)
Comment (murrant) [reply doc/Support/Poller Support.md]: No, each OS that needs it has it explicitly enabled in their OS definition.

It is set to false globally by default and there is no reason for a user to change that.

**Pull request merged**